samples/net/full_mqtt_tls/src/globalsign.inc

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
I wonder if we should document where this came from. This will have to be replaced at some point (it expires in 2021), and Google is free to change who issues their certificate. Their docs currently give a list of certificates (about 60K), which we're not really set up to handle. Having this in a sample is fine, but there will need to be a better way of handling this.

It is also important to emphasize that having this cert is an _absolute requirement_. Using TLS without verifying the certificate is pointless, and arguably worse than not bothering with TLS in the first place. Man-in-the-middle attacks become trivial without it, and there are no security assurances at all about the communication.

Reply:
added some text to the file to indicate the source of the file.  Ideally we'd have a script that populates this, but we can look to that for the future.